cli/src/commit_templater.rs

```diff
@@ -1381,6 +1437,70 @@ impl Template for Vec<Rc<CommitRef>> {
     }
 }
 
+/// Workspace name together with its working-copy commit for templating.
+#[derive(Debug, serde::Serialize)]
+pub struct WorkspaceRef {
```

Comment:
Let's simply make it `Clone` instead of `Rc`. Unlike `CommitRef`, this type doesn't have a shared cache.